java/test/src/main/resources/test_cross_language_invocation.py

```diff
@@ -58,9 +58,7 @@ def py_func_call_java_function():
 @ray.remote
 def py_func_call_java_actor(value):
     assert isinstance(value, bytes)
-    c = ray.cross_language.java_actor_class(
-        "io.ray.test.CrossLanguageInvocationTest$TestActor"
-    )
+    c = ray.cross_language.java_actor_class("io.ray.test.TestActor")
```

Comment:
Why do this change in current PR?

Reply:
The `CrossLanguageInvocationTest.java` includes not only test cases but also the definition parts, which is quite large. We can separate them for clearer. In future we will separate java->cpp and java->python part.